test/test-filter-rewriter.rb

```diff
@@ -110,4 +110,26 @@ def test_under_score
                            ["ref_column.number"]))
     end
   end
+
+  class RegularExpressionTest < self
+    def rewrite(filter, enabled = true)
+      super(filter,
+            :rewrite_regular_expression => enabled)
+    end
+
+    def test_rewrite_one
+      assert_equal("column1 @ \"value1\" &! column2 @ \"value2\"",
+                   rewrite("column1 @ \"value1\" && column2 @~ \"^(?!.*value2).+$\""))
+    end
+
+    def test_no_rewrite
+      assert_equal("column1 @ \"value1\" && column2 @~ \"^(?!.*value2).+$\"",
+                   rewrite("column1 @ \"value1\" && column2 @~ \"^(?!.*value2).+$\"", false))
```

Comment:
Is this needed?
Other options don't test this. Should we test this in all options?

Reply:
Surely, the purpose of this test that can correctly rewrite query or not so that this test is needless.
I'll fix it.